bundles/http/Citrix/DocuSign/metadata.json

```diff
@@ -2,38 +2,44 @@
   "type" : "HTTP",
   "vendor" : "Citrix",
   "id" : "3760ee84-7719-4ac3-ad85-d7364b7063d7",
-  "trackingUuid" : "6501bfff-2fe4-4d42-82f2-d51650043c4f",
+  "trackingUuid" : "61a94043-7a42-4868-b1a6-a0a079c62b07",
   "title" : "DocuSign",
   "description" : "Deploy the Citrix DocuSign integration to quickly and securely make every agreement and approval digital from any device or intranet",
-  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/OVWAgKvpHxhYBqAac82SQhKFn8ssK7gc.png",
-  "masVersion" : "1.71.0",
-  "categories" : ["WEB_SERVICES"],
-  "created" : "2021-03-19T13:11:31",
+  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/C31fG2dN8BSGW3R4psXGxRn6tFUojOoV.png",
+  "masVersion" : "1.81.4.61dfcb45653e8f25b805812340ee1e2d56ae216d",
+  "categories" : [ "INTERNAL_DEVELOPMENT" ],
```

Comment:
Now the Github job bug is fixed, you can change the category back to `WEB_SERVICE`.

Reply:
changes implemented